### PR TITLE
Add tests for MaskedTextBoxTextEditorDropDown.cs

### PR DESCRIPTION
**Pull request by LeafShi1**
Related #10773

## Proposed changes

- Add test file `MaskedTextBoxTextEditorDropDownTests.cs`  for MaskedTextBoxTextEditorDropDown

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12244)

### CODE REVIEW

File: src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/MaskedTextBoxTextEditorDropDownTests.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    [Fact]
+    public void MaskInputRejected_SetsError_WhenInputRejected()
+    {
+        using MaskedTextBox maskedTextBox = new("00：00");
+        using MaskedTextBoxTextEditorDropDown dropDown = new(maskedTextBox);
+        using ErrorProvider errorProvider = dropDown.TestAccessor().Dynamic._errorProvider;
+
+        // No error when setting a correct format value.
+        MaskedTextBox dropDownMaskedTextBox = dropDown.TestAccessor().Dynamic._cloneMtb;
```
Comment (ricardobossan) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/MaskedTextBoxTextEditorDropDownTests.cs:51]: Add `using` here

Comment (ricardobossan) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/MaskedTextBoxTextEditorDropDownTests.cs:51]: Add `using` here

**Review by ricardobossan — COMMENTED** [review]
Other than small comment, LGTM

**Review by lonitra — APPROVED** [review]
Thank you

**Pull request merged**